Xamarin.Forms.Platform.iOS/Renderers/PhoneMasterDetailRenderer.cs

```diff
@@ -318,8 +319,15 @@ void UpdateMasterDetailContainers()
 
 			_detailController.View.AddSubview(detailRenderer.NativeView);
 			_detailController.AddChildViewController(detailRenderer.ViewController);
+
+			SetNeedsStatusBarAppearanceUpdate();
 		}
 
+		public override UIViewController ChildViewControllerForStatusBarHidden()
+		{
+			return (UIViewController)Platform.GetRenderer(((MasterDetailPage)Element).Detail);
```

Comment:
Are we sure there always a detail renderer at this time?
 i wonder if while switching detail pages could have a case where the renderer isn't created yet for the Detail element, this will blow up with NRE. 
Maybe add a check and return base if no renderer is there.